### PR TITLE
Cleanup Packer directory after VHD build

### LINKED ISSUE #3961: 11 GB of packer and rootfs in /home after deployment
Some k8s pre-pulled images are not being cleaned up properly starting in acs-engine 0.22.0.

```
639M    ./home/rootfs-1.9.9
651M    ./home/rootfs-1.10.7
507M    ./home/rootfs-1.8.14
627M    ./home/rootfs-1.7.15
627M    ./home/rootfs-1.7.16
5.0G    ./home/packer
507M    ./home/rootfs-1.8.15
614M    ./home/rootfs-1.11.3
639M    ./home/rootfs-1.9.10
614M    ./home/rootfs-1.11.2
651M    ./home/rootfs-1.10.8
11G     ./home
```

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**What this PR does / why we need it**: Cleanup artifacts in packer dir at the end of VHD build. To avoid having to wait for a new image for the fix, also temporarily do this during CSE provisioning. Also cleanup rootfs- k8s artifacts. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #3961 

**Special notes for your reviewer**:

**If applicable**:
- [ ] documentation
- [ ] unit tests
- [ ] tested backward compatibility (ie. deploy with previous version, upgrade with this branch)

**Release note**:

```release-note
```

### CODE REVIEW

File: parts/k8s/kubernetescustomscript.sh [hunk 1]
```diff
@@ -28,6 +28,9 @@ if [ -f /var/log/azure/golden-image-install.complete ]; then
     FULL_INSTALL_REQUIRED=false
 else
     FULL_INSTALL_REQUIRED=true
+    # temporary fix for cleanup of packer dir
+    # TODO: remove after VHD 0.18.0
+    rm -rf /home/packer
```
Comment (jackfrancis) [inline parts/k8s/kubernetescustomscript.sh]: Shouldn't we keep this in forever just in case?

**Comment by jackfrancis** [comment]
```
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
udev            3.4G     0  3.4G   0% /dev
tmpfs           697M  1.2M  696M   1% /run
/dev/sda1        30G  5.9G   24G  21% /
tmpfs           3.5G     0  3.5G   0% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           3.5G     0  3.5G   0% /sys/fs/cgroup
/dev/sdb1        99G   60M   94G   1% /mnt
/dev/sdc1       252G  185M  239G   1% /var/lib/etcddisk
tmpfs           697M     0  697M   0% /run/user/1001
```

from a master on E2E-generated cluster?

File: parts/k8s/kubernetesinstalls.sh [hunk 2]
```diff
@@ -213,6 +213,7 @@ function extractHyperkube() {
     mv "/usr/local/bin/kubectl-${KUBERNETES_VERSION}" "/usr/local/bin/kubectl"
     chmod a+x /usr/local/bin/kubelet /usr/local/bin/kubectl
     rm -rf /usr/local/bin/kubelet-* /usr/local/bin/kubectl-*
+    rm -rf /home/rootfs-*
```
Comment (jackfrancis) [inline parts/k8s/kubernetesinstalls.sh]: Why is this here?

Comment (CecileRobertMichon) [reply parts/k8s/kubernetesinstalls.sh]: because we want to remove k8s images at provisioning

Comment (jackfrancis) [reply parts/k8s/kubernetesinstalls.sh]: So it's just-in-case? The `mv` commands in `pullHyperkube()` should ensure that stuff doesn't accumulate in `/home/rootfs-*`, right?

Comment (CecileRobertMichon) [reply parts/k8s/kubernetesinstalls.sh]: that won't be applied until a new VHD (we skip pullHyperkube at provisioning if image is already present)

Comment (CecileRobertMichon) [reply parts/k8s/kubernetesinstalls.sh]: That will only be true after we build a new VHD image using the new version of `pullHyperkube` which uses `mv`, not `cp`. Currently with VHD 0.17.0, at provisioning all the `/home/rootfs-*` hyperkube files will be present. This is why I previously had comment 
"remove this with next VHD image".
